src/helpers/initialize.js

```diff
@@ -11,13 +11,14 @@ function initialize({ component, player, playerOpts }) {
   player.on('error', component.eventHandlers.onError);
   player.on('adPlay', component.eventHandlers.onAdPlay);
   player.on('adPause', component.props.onAdPause);
+  player.on('adComplete', component.props.onAdComplete);
+  player.on('adTime', component.eventHandlers.onTime);
```

Comment:
We should have separate handlers for ads vs video time events. Right now it looks like you made them the same?